api/seeds/development/apds.js

```diff
@@ -407,11 +407,44 @@ exports.seed = async knex => {
         end: null,
         name: 'TBD',
         start: null,
-        // totalCost: 64574,
+        totalCost: 64574,
         useHourly: false
       }
     ])
     .returning('id')
+    .then(
+      ([a, b, c, d, e, f, g, h, i, j, k]) =>
+        new Promise(resolve =>
+          knex('activity_contractor_resources_hourly')
+            .insert([
+              { contractor_resource_id: a, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: a, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: b, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: b, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: c, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: c, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: d, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: d, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: e, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: e, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: f, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: f, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: g, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: g, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: h, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: h, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: i, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: i, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: j, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: j, hours: 0, rate: 0, year: 2020 },
+              { contractor_resource_id: k, hours: 0, rate: 0, year: 2019 },
+              { contractor_resource_id: k, hours: 0, rate: 0, year: 2020 }
```

Comment:
The hourly data is required. When an APD is created, the API inserts all this as initial data, but our seed was leaving it out. 😬